src/procedures/procedures.11tydata.yaml

```
@@ -8,6 +8,7 @@ procedureNewZigbee:
         <ol>
           <li>Plug the Home Assistant SkyConnect into the USB extension cable.</li>
           <li>Plug the extension cable into your Home Assistant instance.</li>
+          <li>If you are running Home Assistant in a container, make sure SkyConnect is <a href="https://www.home-assistant.io/installation/linux#exposing-devices" target="_blank">mapped to the container</a>.</li>
```

This could be confusing for some users, as Supervised/OS use containers under the hood as well. We have specific names for the installation methods (see https://www.home-assistant.io/installation/). This specifically is meant for the "Home Assistant Container" method. So how about:

```suggestion
          <li>If you are using the standalone Home Assistant Container installation method make sure SkyConnect is <a href="https://www.home-assistant.io/installation/linux#exposing-devices" target="_blank">mapped to the container</a>.</li>
```